document/core/exec/modules.rst

```diff
@@ -488,10 +488,11 @@ Growing :ref:`memories <syntax-meminst>`
 ..................................
 
 The allocation function for :ref:`modules <syntax-module>` requires a suitable list of :ref:`external values <syntax-externval>` that are assumed to :ref:`match <match-externtype>` the :ref:`import <syntax-import>` vector of the module,
-and a list of initialization :ref:`values <syntax-val>` for the module's :ref:`globals <syntax-global>`.
+a list of initialization :ref:`values <syntax-val>` for the module's :ref:`globals <syntax-global>`,
+and vector of :ref:`function element <syntax-funcelem>` vectors for the module's :ref:`element segments <syntax-elem>`.
```

Comment:
Good point, I should be more consistent. I think I'm using vector only for pieces that are part of the AST and are explicitly specified to be a vector, i.e., limited to 2^32 elements. Otherwise it should be a list or sequence. Fixed.